src/main/java/com/intuit/graphql/orchestrator/schema/transform/FieldResolverTransformerPostMerge.java

```diff
@@ -118,10 +121,31 @@ private FieldResolverContext updateWithTargetFieldData(final FieldResolverContex
     List<ResolverArgumentDefinition> updatedResolverArgumentDefinitions = updateWithInputType(
             resolverDirectiveDefinition.getArguments(), targetFieldInputValueDefinitions, fieldResolverContext);
 
+    FieldContext targetFieldContext = new FieldContext(getParentTypeName(targetFieldDefinition), targetFieldDefinition.getName());
+
+    AtomicReference<String> targetNamespaceRef = new AtomicReference<>();
+
+    //checking if resolver is for a nested field
+    if(StringUtils.contains(resolverDirectiveDefinition.getField(), FQN_FIELD_SEPARATOR)) {
+      String rootField = resolverDirectiveDefinition.getField().split("\\.")[0];
+      targetNamespaceRef.set(
+        Arrays.stream(Operation.values())
+        .map(operation -> new FieldContext(operation.getName(), rootField))
```

Comment:
I think just looking at the rootField might not be enough, since the rootField may have the empty data fetcher for recursive merge. This was the bug that @CNAChino also encountered. Maybe you can call his function somehow. OR even better use the results of his call, so that the function is only called once.